[quarkus-next] Missing comment generated by Liquibase executor in the custom script

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

storage

### Describe the bug

Related to all DB vendors.

After upgrade to Liquibase 4.33.0 the `Create Database Change Log Table` message is missing from the custom-generated script as seen here: https://github.com/liquibase/liquibase/blob/v4.33.0/liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java#L262

However, the log message should be present in the log, so we should be sure the branch is executed.

Not sure if it worth the effort to analyze what has changed since the Liquibase 4.29.1 -> 4.33.0 as the comment is not propagated to the executor and not written to the file. I think it should be only necessary to check if the log message i there.

EDIT: The DATABASECHANGELOG is always created in the database and outside of the custom Liquibase executor, so it seems user's executions are done separately than the default one done by Liquibase.

```
--------- org.keycloak.it.storage.database.dist.PostgreSQLDistTest output end  ---------
Error:  Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 141.591 s <<< FAILURE! - in org.keycloak.it.storage.database.dist.PostgreSQLDistTest
Error: OR] org.keycloak.it.storage.database.dist.PostgreSQLDistTest.testKeycloakDbUpdateScript(CLIResult,RawDistRootPath)  Time elapsed: 18.304 s  <<< FAILURE!
-- java.lang.AssertionError: 
-- 
-- Expected: a string containing "Create Database Change Log Table"
--      but: was "-- *********************************************************************
-- -- * Keycloak database creation script - apply this script to empty DB *
-- -- *********************************************************************
-- 
-- -- *********************************************************************
-- -- Update Database Script
-- -- *********************************************************************
-- -- Change Log: META-INF/jpa-changelog-master.xml
-- -- Ran at: 7/21/25, 12:36 AM
-- -- Against: keycloak@jdbc:postgresql://localhost:32781/keycloak?loggerLevel=OFF
-- -- Liquibase version: 4.33.0
-- -- *********************************************************************
-- 
-- -- Changeset META-INF/jpa-changelog-1.0.0.Final.xml::1.0.0.Final-KEYCLOAK-5461::sthorger@redhat.com
-- CREATE TABLE public.APPLICATION_DEFAULT_ROLES (APPLICATION_ID VARCHAR(36) NOT NULL, ROLE_ID VARCHAR(36) NOT NULL);
-- 
-- CREATE TABLE public.CLIENT (DTYPE VARCHAR(31) NOT NULL, ID VARCHAR(36) NOT NULL, ALLOWED_CLAIMS_MASK BIGINT, ENABLED BOOLEAN DEFAULT FALSE NOT NULL, FULL_SCOPE_ALLOWED BOOLEAN DEFAULT FALSE NOT NULL, NAME VARCHAR(255), NOT_BEFORE INTEGER, PUBLIC_CLIENT BOOLEAN DEFAULT FALSE NOT NULL, SECRET VARCHAR(255), BASE_URL VARCHAR(255), BEARER_ONLY BOOLEAN DEFAULT FALSE NOT NULL, MANAGEMENT_URL VARCHAR(255), SURROGATE_AUTH_REQUIRED BOOLEAN DEFAULT FALSE NOT NULL, DIRECT_GRANTS_ONLY BOOLEAN DEFAULT FALSE NOT NULL, REALM_ID VARCHAR(36));
-- 
-- CREATE TABLE public.CLIENT_SESSION (ID VARCHAR(36) NOT NULL, ACTION INTEGER, CLIENT_ID VARCHAR(36), REDIRECT_URI VARCHAR(255), STATE VARCHAR(255), TIMESTAMP INTEGER, SESSION_ID VARCHAR(36));
-- 
-- CREATE TABLE public.CLIENT_SESSION_ROLE (ROLE_ID VARCHAR(255) NOT NULL, CLIENT_SESSION VARCHAR(36) NOT NULL);
-- 
-- CREATE TABLE public.COMPOSITE_ROLE (COMPOSITE VARCHAR(36) NOT NULL, CHILD_ROLE VARCHAR(36) NOT NULL);
-- 
-- CREATE TABLE public.CREDENTIAL (ID VARCHAR(36) NOT NULL, DEVICE VARCHAR(255), HASH_ITERATIONS INTEGER, SALT BYTEA, TYPE VARCHAR(255), VALUE VARCHAR(255), USER_ID VARCHAR(36));
-- 
-- CREATE TABLE public.EVENT_ENTITY (ID VARCHAR(36) NOT NULL, CLIENT_ID VARCHAR(255), DETAILS_JSON VARCHAR(2550), ERROR VARCHAR(255), IP_ADDRESS VARCHAR(255), REALM_ID VARCHAR(255), SESSION_ID VARCHAR(255), TIME BIGINT, TYPE VARCHAR(255), USER_ID VARCHAR(255));
-- 
-- CREATE TABLE public.FED_PROVIDERS (REALM_ID VARCHAR(36) NOT NULL, USERFEDERATIONPROVIDERS_ID VARCHAR(36) NOT NULL);
```

### Version

quarkus-next (Liquibase 4.33)

### Regression

- [ ] The issue is a regression

### Expected behavior

Tests pass

### Actual behavior

Tests do not pass

### How to Reproduce?

Execute the DB tests

### Anything else?

_No response_

--- a/quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/BasicDatabaseTest.java
+++ b/quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/BasicDatabaseTest.java
@@ -20,7 +20,6 @@ package org.keycloak.it.storage.database;
 import io.quarkus.logging.Log;
 import io.quarkus.test.junit.main.Launch;
 import org.apache.commons.io.FileUtils;
-import org.apache.commons.lang3.StringUtils;
 import org.junit.jupiter.api.MethodOrderer;
 import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
@@ -36,7 +35,6 @@ import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
-import static org.apache.commons.lang3.StringUtils.countMatches;
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.CoreMatchers.is;
@@ -92,11 +90,9 @@ public abstract class BasicDatabaseTest {
         var output = readKeycloakDbUpdateScript(rawDistRootPath);
 
         assertThat(output, notNullValue());
-        assertThat(output, containsString("Create Database Change Log Table"));
-
-        var outputLowerCase = output.toLowerCase();
-        var count = countMatches(outputLowerCase, "create table public.databasechangelog") + countMatches(outputLowerCase, "create table keycloak.databasechangelog");
-        assertThat(count, is(1));
+        assertThat(output, containsString("Keycloak database creation script - apply this script to empty DB"));
+        assertThat(output, containsString("Change Log: META-INF/jpa-changelog-master.xml"));
+        assertThat(output, containsString("Changeset META-INF/jpa-changelog-26.2.6.xml"));
     }
 
     protected static String readKeycloakDbUpdateScript(RawDistRootPath path) {

